netbox/tenancy/migrations/0018_contact_groups.py

```diff
@@ -0,0 +1,94 @@
+# Generated by Django 5.1.5 on 2025-03-11 20:27
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+def migrate_contact_groups(apps, schema_editor):
+    Contacts = apps.get_model('tenancy', 'Contact')
+
+    qs = Contacts.objects.filter(group__isnull=False)
+    for contact in qs:
+        contact.groups.add(contact.group)
+        contact.save()
```

Comment:
I don't think we need to call `save()` on the instance if we're just updating an M2M assignment.